src/index.html

```diff
@@ -10,8 +10,176 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="part">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname:"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name:"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="How old are You?"
+            value="12"
+            min="1"
+            max="100"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="Full date of birth:"
+          >
+        </label>
+
+        <label class="form-field, last" >
+          I accept the term of the agreement:
+          <input
+            type="checkbox"
+            name="I accept the term of the agreement:"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="part">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="E-mail:"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field, last">
+          Password:
+          <input
+            type="password"
+            name="Password:"
+            minlength="7"
+            maxlength="20"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="part">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              name="Do you love cats?"
+              value="Yes"
```

Comment:
it'd be better to use camelCase for values too
```suggestion
              value="yes"
```